reference/tools.rst

```diff
@@ -823,6 +824,13 @@ Fetches a list of all libraries in the package folder.
     - **conanfile** (Required): A `ConanFile` object from which to get the `package_folder`.
     - **folder** (Optional, Defaulted to ``"lib"``): The subfolder where the library files are.
 
+.. warning::
+
+    This tool collects the libraries searching directly inside the directory tree of the package
+    folder. It does not follow any kind of dependency graph, name convention or build order, so it
```

Comment:
=> This tool collects the libraries searching directly inside the package folder, and returns them in no specific order. If the libraries are inter-dependent, then the ``package_info()`` method should order them to achieve correct linking order.